tools/deploy_example_flow.py

```diff
@@ -0,0 +1,83 @@
+"""
+Uploads the "execpropstest" folder as a project to Azkaban at localhost.
+Then executes the flow "job3" with some runtime property overrides.
```

Comment:
Added this script to make it easier to do manual testing locally. I'm not aware of anything similar existing previously.